cloudvolume/volumecutout.py

```diff
@@ -0,0 +1,92 @@
+import os
+
+from six.moves import range
+import numpy as np
+from PIL import Image
+from tqdm import tqdm
+
+from .lib import mkdir
+
+class VolumeCutout(np.ndarray):
+
+  def __new__(cls, buf, dataset_name, layer, mip, layer_type, bounds, *args, **kwargs):
+    return super(VolumeCutout, cls).__new__(cls, shape=buf.shape, buffer=np.ascontiguousarray(buf), dtype=buf.dtype)
+
+  def __init__(self, buf, dataset_name, layer, mip, layer_type, bounds, *args, **kwargs):
+    super(VolumeCutout, self).__init__()
```

Comment:
for python3, just `super()` rather than `super(VolumeCutout, self)` will do the job.

Reply:
Technically CloudVolume is python2 and python3 compatible. If we want to drop Python2 we should make that decision affirmatively.